.github/workflows/ci.yml

```diff
@@ -94,8 +94,9 @@ jobs:
        with:
          version: "v0.7.0"
      - run: |
-         kubectl cluster-info
-         echo "current-context:" $(kubectl config current-context)
+         export CURRENTCONTEXT="$(kubectl config current-context)"
```

Comment:
The project only needs the default Kubernetes cluster. So, I want to give an easy environment for our test cases. I mean to set the environment for all test cases.